tests/provenance/path/path.dl

```diff
@@ -0,0 +1,11 @@
+.pragma "provenance" "1"
```

Comment:
We add license headers to new dl files. e.g. tests/evaluation/magic_goal/magic_goal.dl